redisraft.h

```diff
@@ -251,12 +256,14 @@ typedef struct RedisRaftCtx {
     struct EntryCache *logcache;
     struct RedisRaftConfig *config;     /* User provided configuration */
     bool snapshot_in_progress;  /* Indicates we're creating a snapshot in the background */
+    raft_index_t incoming_snapshot_idx;
```

Comment:
Probably a good idea to document these fields (new and existing ones).

Reply:
Done.